### PR TITLE
Feature/http span

### PR DESCRIPTION
**Pull request by cescoferraro**
This PR implements the new layout for the HTTP span details

## Changes

- Create a new GenericHttpSpanDetail component
-  layout : https://www.figma.com/file/0MPQK2Icv7FvqHJnlColpv/MVP?node-id=632%3A3950

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by cescoferraro** [comment]
@xoscar I have base this branch off my other PR. No need to review this one yet, I have not finished

File: web/src/components/SpanDetail/components/GenericHttpSpanDetail.tsx
Comment (xoscar) [inline web/src/components/SpanDetail/components/GenericHttpSpanDetail.tsx:24]: I think we should move this to the `SpanDetail` facade component as we want to show this for every span.

File: web/src/components/SpanDetail/components/AssetionsSpanComponent.tsx
Comment (xoscar) [inline web/src/components/SpanDetail/components/AssetionsSpanComponent.tsx:31]: I think the empty data style also changed
Screen Shot 2022-05-06 at 8 10 51 a m

File: web/src/components/SpanDetail/components/GenericHttpSpanDetail.tsx
Comment (xoscar) [inline web/src/components/SpanDetail/components/GenericHttpSpanDetail.tsx:37]: Is this a WIP?

File: web/src/components/Trace/Trace.tsx
Comment (xoscar) [inline web/src/components/Trace/Trace.tsx:37]: 🔥

File: web/src/components/Trace/TraceComponent/ResizableDrawer.tsx
Comment (xoscar) [inline web/src/components/Trace/TraceComponent/ResizableDrawer.tsx:1]: I think this should be at the `/components` level as it seems to be generic enough that we can use on different instances outside of the trace

File: web/src/components/Trace/TraceComponent/TraceComponent.tsx
Comment (xoscar) [inline web/src/components/Trace/TraceComponent/TraceComponent.tsx:32]: Should we add styled components?

File: web/src/components/Trace/TraceError/TraceError.tsx
Comment (xoscar) [inline web/src/components/Trace/TraceError/TraceError.tsx:16]: I made some changes to this, please check https://github.com/kubeshop/tracetest/pull/440

**Review by xoscar — CHANGES_REQUESTED** [review]
Hey @cescoferraro GJ, I left some questions and comments. Let me know what you think

**Comment by cescoferraro** [comment]
@xoscar i have pushed the whole feature now
can you review it again? still have some small css issues, but I think this can be solved on a later PR

File: web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx
Comment (xoscar) [inline web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx:30]: I think we should do this either at the model or at the service level. Let's not have business logic at the component level

**Review by xoscar — CHANGES_REQUESTED** [review]
Hey @cescoferraro I left some more comments and concerns. Let me know what you think

File: web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx
Comment (cescoferraro) [reply web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx:6]: sure

File: web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx
Comment (cescoferraro) [reply web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx:16]: sure! we need to write down those standards

File: web/src/components/Trace/TraceComponent/TimelineChart.tsx
Comment (cescoferraro) [reply web/src/components/Trace/TraceComponent/TimelineChart.tsx:228]: i have not changed anything, just added the variable to the dependencie array
because eslint was complaining about this

File: web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx
Comment (cescoferraro) [reply web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx:29]: done

**Comment by xoscar** [comment]
Hey @cescoferraro I think you should have waited for my approval before merging, there are still some comments that I had based on your latest changes.

Let's try to wait for the final approval form the reviewer (if there are requested changes) before merging 😬

**Pull request merged**
